### ISSUE TITLE
Our website taking a hit

### ISSUE THREAD

**Issue by randell** [AusDTO/disco_crawl#30]
Hi guys, would it be possible to temporarily disable crawling www.hpca.nsw.gov.au? Our website is taking a hit at the moment. We can let you know when the site is ready again for crawling if that helps. Thanks!

**Comment by nokout** [comment]
Hi @randell - you should have recieved some feedback through other channels but we have stopped the crawler until we clarify the issue. Cheers, Nigel

**Comment by gordjw** [comment]
Hi @randell, thanks for letting us know, would you mind dropping me a line at my email (in my profile) so we can talk about the traffic you've been seeing?

**Comment by randell** [comment]
Thanks, guys. We have yet to configure our website to handle the traffic. We'll let you know as soon as there are any updates.

**Comment by monkeypants** [comment]
we still leaving your site alone, but one day somebody might forget why and stop, or someone else might use this code with different exclusions, so it should be viewed as a temporary situation.

Are you able to add a `robots.txt` to your site @randell ? I'd like to be able to close this ticket.

**Comment by koriaf** [comment]
Yes, please note that you are able to define timeout for your website crawls in the `robots.txt` - something like
```
User-agent: *
Crawl-delay: 30
```
Which should help in your case.

**Comment by randell** [comment]
Thanks, guys. Yes, I can add those lines in our robots.txt.
